config.rb

```diff
@@ -110,7 +110,7 @@
   content.gsub(%r{^<(h[0-6])(.*?)</\1>}m) do |header_text|
     header = Nokogiri::HTML::DocumentFragment.parse(header_text).children.first
     id = header.attr(:id)
-    header.children.before("<a class='anchor' href='##{id}'></a>") if id
+    header.children.before("<a class='anchor' href='##{id}'><span class='visuallyhidden'>#{header.text} permalink</a>") if id 
     header.to_html
```

Comment:
The span is not closed. Also (and sorry if this was talked about already and I'm koolaid-manning into this but) would it be better to put the assistive class on the anchor?

Reply:
Ah, thanks! Added the closing tag. (Legit laughed at Koolaidmaning it)

From my understanding, the `a` is what allows the `:hover` that shows the 🔗. So we don't want to visually hide that, just the descriptive text for SRs who would otherwise see an empty link without. Hopefully that makes sense, and open to hearing if I misunderstood the intent/execution of the `a` tag.

Reply:
ah ok cool

Reply:
I think git/hub lost your most recent changes that addressed these resolved comments.